inst/qml/processCapabilityStudies.qml

```diff
@@ -238,6 +205,45 @@ Form
 						decimals:				7
 					}
 				}
+
+				CheckBox
+				{
+					name: 						"CapabilityStudyPlot"
+					label: 						qsTr("Process capability plot")
+					checked: 					true
+					enabled:          checkedupper.checked | checkedtarget.checked | checkedlower.checked
```

Comment:
```suggestion
					enabled:          checkedupper.checked || checkedtarget.checked || checkedlower.checked
```
since this is QML, `|` (bitwise or) is even more different than `||` (logical or). For example, `1 | 2 == 3` whereas `1 || 2 == true`.